libs.versions.toml

```diff
@@ -7,7 +7,7 @@ androidx-appcompat = "1.4.0"
 androidx-arch-core = "2.0.1"
 androidx-browser = "1.4.0"
 androidx-cardview = "1.0.0"
-androidx-compose = "1.1.1"
+androidx-compose = "2024.08.00"
```

Comment:
> Since the compiler has been [moved](https://android-developers.googleblog.com/2024/04/jetpack-compose-compiler-moving-to-kotlin-repository.html) to be part of Kotlin itself, I didn't declare a version for it on the catalog, which means we'll keep using a [literal](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/294ab2417061836956e98f44c15bc20e80043057/example/build.gradle#L74) version in FluxC temporarily, please share your thoughts on this.

Suggestion (💡): I suggest adding the `Compose Compiler` version here too, and then using that where necessary, both on the `example` app of this project and `FluxC` itself. I understand that Kotlin 2.0 will make this redundant, but unfortunately, until we get there (don't know when exactly), we would still need to use that.

- Definition: `androidx-compose-compiler = "1.5.15"`
- Usage:

```
    composeOptions {
        kotlinCompilerExtensionVersion = libs.versions.androidx.compose.compiler.get()
    }
```

Reply:
Thanks @ParaskP7, this is done in d5de244f8db10d0e350c968da0c5ebb263e18590

Reply:
👋 @hichamboushaba and 🫣 I just checked the one commit you commented on and didn't go checking all commits again, apologies for the wait, everything is looking great, let's proceed with the merge! 🎉

Reply:
Yeah, sorry for not mentioning both commits 😄